AutomatedTesting/Gem/PythonTests/atom_renderer/atom_hydra_scripts/hydra_GPUTest_LightComponent.py

```diff
@@ -0,0 +1,275 @@
+    azlmbr.components.TransformBus(azlmbr.bus.Event, "SetLocalRotation", area_light.id, area_light_rotation)
+    general.idle_wait(1.0)
+    screenshot_utils.take_screenshot_game_mode("AreaLight_4", area_light_entity_name)
+
+    # Swap the "Spot (disk)" light type to the "Point (sphere)" light type and take screenshot.
+    sphere_light_type = atom_component_helper.LIGHT_TYPES['sphere']
+    azlmbr.editor.EditorComponentAPIBus(
+        azlmbr.bus.Broadcast,
+        'SetComponentProperty',
+        light_component_id_pair,
+        light_type_property,
+        sphere_light_type
+    )
+    general.idle_wait(1.0)
+    screenshot_utils.take_screenshot_game_mode("AreaLight_5", area_light_entity_name)
+
+    editor.ToolsApplicationRequestBus(bus.Broadcast, "DeleteEntityById", area_light.id)
+
+
+def spot_light_test():
+    """
+    Basic test for the Light component attached to a "spot_light" entity.
+    Example: Entity addition/deletion & enters game mode w/ Light component attached using various light types.
+    """
+    # Disable "Directional Light" component for the "directional_light" entity
+    directional_light_entity_id = hydra.find_entity_by_name("directional_light")
+    directional_light = hydra.Entity(name='directional_light', id=directional_light_entity_id)
+    directional_light_component_type = azlmbr.editor.EditorComponentAPIBus(
+        azlmbr.bus.Broadcast, 'FindComponentTypeIdsByEntityType', ["Directional Light"], 0)[0]
+    directional_light_component = azlmbr.editor.EditorComponentAPIBus(
+        azlmbr.bus.Broadcast, 'GetComponentOfType', directional_light.id, directional_light_component_type
+    ).GetValue()
+    editor.EditorComponentAPIBus(bus.Broadcast, "DisableComponents", [directional_light_component])
+    general.idle_wait(0.5)
+
+    # Disable "Global Skylight (IBL)" and "HDRi Skybox" components for the "global_skylight" entity
+    global_skylight_entity_id = hydra.find_entity_by_name("global_skylight")
+    global_skylight = hydra.Entity(name='global_skylight', id=global_skylight_entity_id)
+    global_skylight_component_type = azlmbr.editor.EditorComponentAPIBus(
+        azlmbr.bus.Broadcast, 'FindComponentTypeIdsByEntityType', ["Global Skylight (IBL)"], 0)[0]
+    global_skylight_component = azlmbr.editor.EditorComponentAPIBus(
+        azlmbr.bus.Broadcast, 'GetComponentOfType', global_skylight.id, global_skylight_component_type
+    ).GetValue()
+    editor.EditorComponentAPIBus(bus.Broadcast, "DisableComponents", [global_skylight_component])
+    hdri_skybox_component_type = azlmbr.editor.EditorComponentAPIBus(
+        azlmbr.bus.Broadcast, 'FindComponentTypeIdsByEntityType', ["HDRi Skybox"], 0)[0]
+    hdri_skybox_component = azlmbr.editor.EditorComponentAPIBus(
+        azlmbr.bus.Broadcast, 'GetComponentOfType', global_skylight.id, hdri_skybox_component_type
+    ).GetValue()
+    editor.EditorComponentAPIBus(bus.Broadcast, "DisableComponents", [hdri_skybox_component])
+    general.idle_wait(0.5)
```

Comment:
nit: might be easier to just make these entities invisible than to have to find individual components and disable them.

Reply:
for now i am going to keep it as disable but i will keep this in mind for future updates

Reply:
I think we should keep it as disable since using visible, the component would still be active and there is an odd chance that it may impact the scene. entity visible status may only be in editor mode. screenshots enter game mode.